apps/ios/Shared/Views/UserSettings/UserProfilesView.swift

```diff
@@ -9,57 +9,90 @@ import SimpleXChat
 struct UserProfilesView: View {
     @EnvironmentObject private var m: ChatModel
     @Environment(\.editMode) private var editMode
-    @State private var selectedUser: Int? = nil
-    @State private var showAddUser: Bool? = false
+    @State private var alert: UserProfilesAlert?
+
+    private enum UserProfilesAlert: Identifiable {
+        case deleteUser(index: Int)
+        case error(title: LocalizedStringKey, error: LocalizedStringKey = "")
+
+        var id: String {
+            switch self {
+            case let .deleteUser(index): return "deleteUser \(index)"
+            case let .error(title, _): return "error \(title)"
+            }
+        }
+    }
 
     var body: some View {
         List {
-            Section("Your profiles") {
-                ForEach(Array(m.users.enumerated()), id: \.0) { i, userInfo in
-                    userProfileView(userInfo, index: i)
-                        .deleteDisabled(userInfo.user.activeUser)
+            Section {
```

Comment:
There's a following bug: if you change a user (switch or create a new one), then Edit users list without closing the view, the initial active user cannot be deleted.

Regardless, maybe we should close settings view after changing user?